src/components/NavGlobal.tsx

```diff
@@ -1,31 +1,98 @@
+import classnames from 'classnames';
 import Image from 'next/image';
 import Link from 'next/link';
+import React from 'react';
 
 import CloudGovLogo from '@/components/svgs/CloudGovLogo';
 import dashboardIcon from '@/../public/img/logos/dashboard-icon.svg';
 import cloudPagesIcon from '@/../public/img/logos/cloud-pages-icon.svg';
 
 import { Auth } from '@/components/auth/Auth';
 
+const sharedLinkClasses = [
+  'border-bottom-05',
+  'border-top-0',
+  'border-transparent',
+  'border-x-0',
+  'flex-align-center',
+  'margin-right-205',
+  'padding-bottom-1',
+  'padding-top-105',
+  'desktop:padding-bottom-2',
+  'desktop:padding-top-205',
+  'text-no-underline',
+];
+
+function LinkProduct({
+  active = false,
+  children,
+  className,
+  href,
+}: {
+  active?: boolean;
+  children: React.ReactNode;
+  className: string;
+  href: string;
+}) {
+  const classes = classnames(
+    'display-flex',
+    'margin-bottom-2px',
+    'text-no-underline',
+    'text-white',
+    {
+      'border-white': active,
+      'text-bold': active,
+    },
```

Comment:
I think they need to be because they toggle depending on whether active is true or false.  I was following the example set in the uswds components we adapted from Truss, but it also looks like that's how it's done in the documentation as well:  https://github.com/JedWatson/classnames?tab=readme-ov-file#usage

Reply:
TIL, thanks!